frontend/webpack.config.js

```diff
@@ -12,7 +12,7 @@ var path = require('path');
 var config = {
     entry: path.join(__dirname, 'app', 'index.js'),
     output: {
-        path: path.resolve(__dirname, 'public'),
+        path: '/var/www/promis', // TODO: do we need to parametrise this?
```

Comment:
No, this script is run at `/usr/src/promis`, not at `/var/www/promis` which should only contain ouptut. Also why do we need `public` directory in first place? 

My reasoning here:
- `/usr/src/promis` ⇒ source React files
- `/var/www/promis` ⇒ compiled JS/HTML/whatev that is served by nginx, no development files there

Is that correct?